### ISSUE TITLE
what do you think about this way to recognize screenshots?

### ISSUE THREAD

**Issue by haemi** [clayallsopp/ShotBlocker#1]
http://blog.chpwn.com/post/38491252231

> On a technical level, the two basic pieces are UILongPressGestureRecognizer (or -touchesCancelled:withEvent:, if you want) and UIApplicationDelegate.

**Comment by clayallsopp** [comment]
Yeah, it works if "anything that you want to protect will require you to have the user to touch the screen to see."

There might be a way to support this (like `[[ShotBlocker sharedManager] addDetectionGestureToView: view]`), but it's not a one-size-fits-all method =\ would welcome a PR for it!

**Comment by drakos** [comment]
Is there a way to detect screenshot before it's saved in camera roll? Like I want to alter the views before screenshot is taken.

Cheers!

**Comment by clayallsopp** [comment]
Nope, all screen shot detection techniques only work after-the-fact.

**Comment by drakos** [comment]
Thanks! Great work!
